recipes/plasflow/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# install PlasFlow
+$PYTHON -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+
+# install models
+git clone https://github.com/smaegol/PlasFlow.git
```

Comment:
Can we checkout a specific commit?

Reply:
[Define metedata from multiple sources](https://github.com/conda/conda-docs/blob/f5eb0fe9206809216b08100d07249bfc970a4098/docs/source/user-guide/tasks/build-packages/define-metadata.rst#source-from-multiple-sources)

```yaml
source:
  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
    sha256: 352d5d7cd73672130e32f3f61c3f2ad95aa684d7aebf7668ab3f2594996c338b
  - url: https://github.com/smaegol/PlasFlow/archive/v1.1.tar.gz
    sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
    folder: PlasFlow
```